client/ayon_traypublisher/plugins/create/create_editorial_simple.py

```diff
@@ -204,6 +204,8 @@ def apply_settings(self, project_settings):
         )
         creator_settings = editorial_creators.get(self.identifier)
 
+        self.enabled = creator_settings.get("enabled", True)
```

Comment:
If it's in settings, you don't have to do it safely.
```suggestion
        self.enabled = creator_settings["enabled"]
```